source/extensions/tracers/opencensus/opencensus_tracer_impl.cc

```diff
@@ -236,6 +237,8 @@ void Span::injectContext(Tracing::TraceContext& trace_context, const Tracing::Up
   }
 }
 
+std::string Span::getSpanIdAsHex() const { return std::string{}; }
```

Comment:
Prefer returning EMPTY_STRING.

Reply:
Thanks, fixed that [here](https://github.com/envoyproxy/envoy/pull/33909/commits/2f9767d7f6f7edbbcacd148a2efc31953b87df2b#diff-e70df531b553c85beb87a955c2f42c81df544610d09492b4037802c4a85d34b6R240).